### PR TITLE
Blog pagination

### PR DESCRIPTION
**Pull request by RobinLinde**
Adds pagination for blog, renames `visable` to `visible`

### CODE REVIEW

**Comment by wjtje** [comment]
I changed a few things, see commit

File: src/hooks.server.ts [hunk 1]
```diff
@@ -34,6 +34,14 @@ export const handle: Handle = async ({ event, resolve }) => {
 			});
 		}
 
+		// Redirect blog to first page
+		if (pathname.endsWith('/blog') || pathname.endsWith('/blog/')) {
+			return new Response(undefined, {
```
Comment (wjtje) [inline src/hooks.server.ts:39]: This is not how you should be doing a redirect here, see: https://kit.svelte.dev/docs/load#redirects

File: src/routes/+layout.svelte [hunk 2]
```diff
@@ -22,7 +22,7 @@
 			name: $t('navigation.projects')
 		},
 		{
-			path: `blog`,
+			path: `blog/1`,
```
Comment (wjtje) [inline src/routes/+layout.svelte:25]: Maybe if you change the redirect, this can be changed back

File: src/hooks.server.ts (hunk 1 above)
Comment (RobinLinde) [reply src/hooks.server.ts:39]: Maybe we can also change the one at line 31 https://github.com/wjtje/wjtje.dev/blob/b775b6fa77d226ffd405f17c935b0fdbbb5bd31e/src/hooks.server.ts#L31-L34

**Review by wjtje — APPROVED** [review]

**Pull request merged**